ckanext/scheming/tests/test_validation.py

```diff
@@ -80,6 +80,35 @@ def test_date_field_in_resource(self):
                 'camels_in_photo': 2,
                 'date': '2015-01-01'}])
 
+class TestDateTimes(object):
+    def test_datetime_field_rejects_non_isodates(self):
+        lc = LocalCKAN()
+        assert_raises(ValidationError, lc.action.package_create,
+            type='camel-photos',
+            name='fred',
+            a_relevant_datetime='31/11/2014',
+        )
+        assert_raises(ValidationError, lc.action.package_create,
```

Comment:
these should be in separate tests (same goes for the date tests above) and how about at least one test of a valid datetime?